ste/mgr-JobMgr.go

```diff
@@ -754,10 +804,15 @@ func (jm *jobMgr) poolSizer() {
 		} else if actualConcurrency > targetConcurrency {
 			hasHadTimeToStablize = false
 			jm.poolSizingChannels.scalebackRequestCh <- struct{}{}
+		} else if actualConcurrency == 0 && targetConcurrency == 0 {
+			fmt.Println("Exiting Pool sizer")
+			return
 		}
```

Comment:
See L830, we should protect that also with this check and don't do anything, else it may again set targetConcurrency after it's set to 0 once.
And we need to be sure that we will never enter this function with targetConcurrency==0